website/src/controllers/api-v3/groups.js

```diff
@@ -4,14 +4,15 @@ import _ from 'lodash';
 import cron from '../../middlewares/api-v3/cron';
 import { model as Group } from '../../models/group';
 import { model as User } from '../../models/user';
+import { model as EmailUnsubscription } from '../../models/emailUnsubscription';
 import {
   NotFound,
   BadRequest,
   NotAuthorized,
 } from '../../libs/api-v3/errors';
 import * as firebase from '../../libs/api-v3/firebase';
-import { txnEmail } from '../../libs/api-v3/email';
-// import { encrypt } from '../../libs/api-v3/encryption';
+import { sendTxn } from '../../libs/api-v3/email';
```

Comment:
Let's  make this `import { sendTxn as sendTxnEmail }` so it's clear what this function is doing when we use it.